### PR TITLE
split useNativeTokenBalance

### PR DESCRIPTION
**Pull request by smartdevcom**
Mocha test is passed in the SDK. cc @peetzweg

### CODE REVIEW

File: packages/sdk/src/modules/FundOperations.ts [hunk 1]
```diff
@@ -80,17 +80,9 @@ export function withFundOperations<TBase extends MidasBaseConstructor>(Base: TBa
       return await this.mint(cTokenAddress, amount);
     }
 
-    async repay(cTokenAddress: string, underlyingTokenAddress: string, isRepayingMax: boolean, amount: BigNumber) {
+    async repayBorrow(cTokenAddress: string, isRepayingMax: boolean, amount: BigNumber) {
```
Comment (peetzweg) [inline packages/sdk/src/modules/FundOperations.ts:83]: Why do we now have two repay functions?
`repayBorrow` & `repay`?

Aren't we always repaying a borrowed token?

**Comment by peetzweg** [comment]
What is changed in this PR again? A bit confused.

File: packages/sdk/src/modules/FundOperations.ts (hunk 1 above)
Comment (smartdevcom) [reply packages/sdk/src/modules/FundOperations.ts:83]: `repay` = `approved` + `repayBorrow`. So `repay` has 2 steps for transactions, it needs to be separated to show tx to users transparently

**Comment by smartdevcom** [comment]
> What is changed in this PR again? A bit confused.

As I said before, I pushed the codes which show transaction steps for supply directly to `main` with my fault, so after that I made pr and implemented for `repay`, that is PR for that

**Pull request closed without merging**